corehq/apps/commtrack/tests/util.py

```diff
@@ -1,26 +1,28 @@
 from django.test import TestCase
+from lxml import etree
+
 from casexml.apps.case.tests import delete_all_cases, delete_all_xforms
 from casexml.apps.case.tests.util import delete_all_sync_logs
 from casexml.apps.case.xml import V2
 from casexml.apps.phone.tests.utils import generate_restore_payload
-from casexml.apps.stock.models import StockReport, StockTransaction
 from casexml.apps.stock.const import COMMTRACK_REPORT_XMLNS
-from corehq.apps.commtrack import const
+from casexml.apps.stock.models import StockReport, StockTransaction
+from couchforms.models import XFormInstance
+from dimagi.utils.couch.database import get_safe_write_kwargs
+
+from corehq.apps.domain.models import Domain
+from corehq.apps.domain.shortcuts import create_domain
 from corehq.apps.groups.models import Group
 from corehq.apps.locations.models import Location, LocationType
-from corehq.apps.domain.shortcuts import create_domain
-from corehq.apps.domain.models import Domain
-from corehq.apps.commtrack.sms import StockReportParser, process
-from corehq.apps.commtrack.util import get_default_requisition_config, \
-    bootstrap_location_types
-from corehq.apps.commtrack.models import SupplyPointCase, CommtrackConfig, ConsumptionConfig
-from corehq.apps.users.models import CommCareUser
-from corehq.apps.sms.backend import test
-from corehq.apps.commtrack.helpers import make_supply_point
 from corehq.apps.products.models import Product
-from couchforms.models import XFormInstance
-from dimagi.utils.couch.database import get_safe_write_kwargs
-from lxml import etree
+from corehq.apps.sms.backend import test
+from corehq.apps.users.models import CommCareUser
+
+from .. import const
```

Comment:
I really like the following pattern for imports:
```
# python standard library stuff

# django imports

# other 3rd party imports

# imports from other modules in this project

# explicit relative imports from this same module
```
With each block of imports alphabetized.  It's basically a progression from generic to specific.  Doing the explicit relative imports to me shows more clearly that these are imports from the same module - it's a way to see the cross-linking at a glance.  I won't always split these up quite as much, sometimes it'll be more like this:
```
# all 3rd party and python imports

# imports from other modules

# module imports
```
It depends on how many imports there are.  In this case, I know I'm gonna be doing a lot of commtrack work, so I arranged stuff to be less annoying.